### PR TITLE
Add test scenarios involving ursine packages and default streams

### PR DESCRIPTION
**Pull request by r4f4**
Signed-off-by: Rafael dos Santos <rdossant@redhat.com>

### CODE REVIEW

File: dnf-docker-test/features/module-install-pkg-2.feature [hunk 1]
```diff
@@ -12,3 +12,29 @@ Feature: Installing package from module - error handling
        Then rpmdb does not change
         And the command stderr should match regexp "Error: Unable to find a match"
         And the command exit code is 1
+
```
Comment (kkaarreell) [inline dnf-docker-test/features/module-install-pkg-2.feature:15]: could you please add @setup here.

File: dnf-docker-test/features/module-install-pkg-3.feature [hunk 2]
```diff
@@ -27,3 +28,26 @@ Feature: Installing package when default stream is defined
           | version | -1    |
           | enabled | 1     |
           | stream  | f26   |
+
+  Scenario: cleanup from previous scenario
+      Given I successfully run "dnf remove TestY -y"
+        And I successfully run "dnf module disable ModuleY"
+
+  Scenario: rpm from enabled stream is preferred regardless of NVRs
```
Comment (kkaarreell) [inline dnf-docker-test/features/module-install-pkg-3.feature:37]: Given that you have default stream still defined how is this scenario different from the previous one?
I think you should rather enable f27 (non-default) stream here.

File: dnf-docker-test/features/module-install-pkg-3.feature [hunk 3]
```diff
@@ -27,3 +28,26 @@ Feature: Installing package when default stream is defined
...
+        And I successfully run "dnf install -y TestY"
+       Then rpmdb changes are
+          | State     | Packages       |
+          | installed | TestY/1-1.modY |
+
+  Scenario: cleanup from previous scenario
+      Given I successfully run "dnf remove TestY -y"
+        And I successfully run "dnf module disable ModuleY"
+
+  Scenario: rpm from default stream is preferred regardless of NVRs
```
Comment (kkaarreell) [inline dnf-docker-test/features/module-install-pkg-3.feature]: I believe this is the same as you first scenario
Scenario: I can install a specific package from a module not enabled when default stream is defined

File: dnf-docker-test/features/module-install-pkg-2.feature (hunk 1 above)
Comment (r4f4) [reply dnf-docker-test/features/module-install-pkg-2.feature:15]: Sure

File: dnf-docker-test/features/module-install-pkg-3.feature (hunk 2 above)
Comment (r4f4) [reply dnf-docker-test/features/module-install-pkg-3.feature:37]: Ok, I'll make the change to f27 stream.

File: dnf-docker-test/features/module-install-pkg-3.feature (hunk 3 above)
Comment (r4f4) [reply dnf-docker-test/features/module-install-pkg-3.feature]: You are right. I'll remove it.

**Pull request merged**